open_mastr/soap_api/download.py

```diff
@@ -395,7 +395,22 @@ def to_csv(df, technology):
 
     csv_file = os.path.join(data_path, filenames["raw"][technology]["joined"])
 
-    df.to_csv(csv_file, index=True, index_label="EinheitMastrNummer", encoding="utf-8")
+    if os.path.exists(csv_file):
+        df.to_csv(
+            csv_file,
+            mode="a",
```

Comment:
The new method with 'append' leads to error, if the user tries to export the same tech for the second time, since the table already exists. Is the use case in this manner is intended?

Reply:
I have not considered this to be a use case. In my view, the deletion of tables is a good way to avoid having exported csv tables from different database versions.